nixos/lib/test-driver/test-driver.py

```diff
@@ -19,6 +20,7 @@
 import tempfile
 import time
 import unicodedata
+import io
```

Comment:
i see `codecs` being used later but not `io` - is this implicitly used or does its initialization do some side effects?

Reply:
Ha, thats a leftover from a previous try with IO.

Reply:
Removed.